src/hooks/sdk/docs/useMediaStreamMetrics.stories.mdx

```diff
@@ -0,0 +1,62 @@
+<Meta title="SDK Hooks/useMediaStreamMetrics" />
+
+# useMediaStreamMetrics
+
+The `useMediaStreamMetrics` hook returns the available audio, video and bandwidth metrics for the current meeting session. Video metrics contains stream metrics for all attendeeIds.
+
+This hook re-renders frequently. You should avoid using it high in your app tree, and ideally use it in your leaf components.
+
+### Return Value
+
+```javascript
+{
+  audioPacketsSentFractionLossPercent: number | null; 
+  audioPacketsReceivedFractionLossPercent: number | null;
+  availableOutgoingBandwidth: number | null;
+  availableIncomingBandwidth: number | null;
+  videoStreamMetrics: { [attendeeId: string]: { [ssrc: string]: {[key: string]: number} } }
```

Comment:
You should list the names of all the videoStreamMetric we expose too.

Reply:
How should I do that?

Reply:
You should know the name of the metrics we expose from your changes in JS SDK right?

Reply:
How do I do it in the interface? Do we have an example?

Reply:
This is not resolved right? We can get the stats shown from metric maps in JS SDK as Trung is saying right?

https://github.com/aws/amazon-chime-sdk-js/blob/master/src/clientmetricreport/DefaultClientMetricReport.ts#L208

OR you can use metric names from the `VideoMetrics` interface right?

Reply:
Gotcha. Will udpate